DataBaseConnection.obj : error LNK2019: unresolved external symbol __imp__invalid_parameter referenced in function "void __cdecl std::_Adjust_manually_vector_aligned(void * &,unsigned __int64 &)" (?_Adjust_manually_vector_aligned@std@@YAXAEAPEAXAEA_K@Z)
DataCon1.1.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
sha256.obj : error LNK2001: unresolved external symbol __imp__invalid_parameter
DataBaseConnection.obj : error LNK2019: unresolved external symbol __imp__CrtDbgReport referenced in function "void __cdecl std::_Adjust_manually_vector_aligned(void * &,unsigned __int64 &)" (?_Adjust_manually_vector_aligned@std@@YAXAEAPEAXAEA_K@Z)
DataCon1.1.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
sha256.obj : error LNK2001: unresolved external symbol __imp__CrtDbgReport
msvcprtd.lib(locale0_implib.obj) : error LNK2019: unresolved external symbol __imp__free_dbg referenced in function "public: static void __cdecl std::_Fac_node::operator delete(void *)" (??3_Fac_node@std@@SAXPEAX@Z)
msvcprtd.lib(locale0_implib.obj) : error LNK2019: unresolved external symbol __imp__malloc_dbg referenced in function "public: static void * __cdecl std::_Fac_node::operator new(unsigned __int64)" (??2_Fac_node@std@@SAPEAX_K@Z)
D:\_MyRepos\Networking\DataBaseConnection\x64\Debug\DataBaseConnection.exe : fatal error LNK1120: 4 unresolved externals
